### ISSUE TITLE
Methods added to interfaces are not reported as source incompatibility if interface did not contain methods in old version

### ISSUE THREAD

**Issue by SgtSilvio** [lvc/japi-compliance-checker#54]
Old version
```java
interface A {
}
```
New version
```java
interface A {
    void test();
}
```
